action.yml

```diff
@@ -48,6 +48,12 @@ inputs:
     description: |-
       Set headers (object metadata).
     required: false
+  process_gcloudignore:
+    description: |-
+      Process the .gcloudignore file in the root of the repository and do not
```

Comment:
Should we consider that a gcloudignore maybe at [path input](https://github.com/google-github-actions/upload-cloud-storage/blob/d558623474cdac86d16a44ea3ed19c89eb5e4671/action.yml#L12) instead rather than at repo root?

Reply:
The way `gcloud alpha storage` works is that it only parses it in the directory where the gcloud command is run (even if you're uploading a subdir). I was trying to mirror that behavior.